lib/liquid/standardfilters.rb

```diff
@@ -432,6 +432,10 @@ def default(input, default_value = '')
 
     private
 
+    def context
```

Comment:
We could simply add `attr_reader :context`.

`||= nil` is unnecessary (the method could simply return `@context`). When the variable is `nil` (or `false`), `||=` will keep re-evaluating/re-assigning the value. To avoid that we'd need to use `defined?`.

Reply:
Thanks.